packages/apollo/src/utils/lists.ts

```diff
@@ -67,26 +105,58 @@ export class ListHelper {
             this.skipCacheOnRead = options.cacheOptions.skipOnRead
         }
 
-        bindAll(this, '_readPage', '_networkOnlyRead', 'getReadFunction', 'mergeLists')
+        bindAll(this, '_readSlicePage', '_readAllList', '_networkOnlyRead', 'getReadFunction', 'mergeLists')
```

Comment:
Why do you need to rename?

`_readPage` and `_readAll` is simple and good enough I guess

Reply:
page and slice are basically the same word. So it's like `_readPagePage` or `_readSliceSlice`...

I prefer page imho